app/controllers/search_controller.rb

```diff
@@ -8,7 +8,7 @@ def search
                        .paginate(page: params[:page], per_page: 25)
 
                if search_data[:search].present?
-                 posts.search(search_data[:search]).user_sort({ term: params[:sort], default: :search_score },
+                 posts.search(search_data[:search]).user_sort({ term: params[:sort] },
```

Comment:
Could do with updating to re-add a default for the new weighted search, and to change the `relevance` sort to the same weighted search. Not sure whether our implementation of `user_sort` supports that via `Arel.sql`, though, so that's probably not one for this PR - needs investigation.